modules/table/build.gradle

```diff
@@ -64,6 +64,7 @@ dependencies {
     testImplementation project(':ignite-page-memory')
     testImplementation project(':ignite-storage-rocksdb')
     testImplementation project(':ignite-placement-driver-api')
+    testImplementation project(':ignite-placement-driver')
```

Comment:
Curious why placement-driver-api is not enough?

Reply:
Because I need the `Lease` class